frontend/www/sass/base/_variables.scss

```
@@ -113,6 +135,25 @@
   // CLARIFICATIONS
   --clarifications-list-pre-font-color: #333333;
   --clarifications-list-pre-background-color: #f5f5f5;
+  --clarification-resolved-font-color: #468847;
+  --clarification-resolved-background-color: #dff0d8;
+  --clarification-resolved-gradient-from-background-color: rgb(223, 240, 216) 0;
```

```suggestion
  --clarification-resolved-gradient-from-background-color: rgb(223, 240, 216);
```

el 0 / 100% es innecesario en los gradientes (es el default)

---

Listo